tests/expressions/tests.py

```diff
@@ -332,12 +336,195 @@ def test_object_update_unsaved_objects(self):
         with self.assertRaisesMessage(ValueError, msg):
             acme.save()
 
+    def test_ticket_22288_in_f_integer(self):
```

Comment:
I had been taking my lead from other functions in that class, like test_ticket_18375_chained_filters and test_ticket_18375_kwarg_ordering_2. I'll switch mine.